docs/running-in-ci.md

```diff
@@ -0,0 +1,239 @@
+# Running in CI
+
+[TOC]
+
+The following instructions use [Semgrep CI](https://github.com/returntocorp/semgrep-action) and require a free [Semgrep Community](https://semgrep.dev/manage) or paid Semgrep Team account. `SEMGREP_DEPLOYMENT_ID` and `SEMGREP_TOKEN` information is available at [Manage > Settings](https://semgrep.dev/manage/settings) after login.
+
+!!! danger
+    `SEMGREP_TOKEN` is a secret value: DO NOT HARDCODE IT AND LEAK CREDENTIALS. Use your CI provider's secret or environment variable management feature to store it.
+
+Semgrep can seamlessly integrate into your CI pipeline using GitHub Actions or GitLab CI.
+
+## GitHub Actions
+
+The easiest way to set up the Semgrep CI integration in GitHub
+is via the [Semgrep Community](https://semgrep.dev/manage/) app.
+
+When you install Semgrep on a GitHub organization,
+you select which repositories should be visible to Semgrep.
+The ones you select will appear on the [Projects page](https://semgrep.dev/manage/projects).
+
+!!! info
+    You can update this list of selected repositories at any time
+    through your organization's settings page on GitHub.
+    Just go to Settings > Installed GitHub Apps > semgrep.dev > Configure
+    and make your changes in the 'Repository access' section.
+  
+To set up Semgrep CI in GitHub Actions on one of these projects,
```

Comment:
@underyx Taking this in a different direction, what do you think of providing problem-focused instructions that place auto project-setup instructions under something more like "Rolling out Semgrep at your org" or "Adding Semgrep to many projects"?